pkg/provider/azure_standard.go

```diff
@@ -296,8 +296,8 @@ func getBackendPoolName(clusterName string, isIPv6 bool) string {
 // getBackendPoolNames returns the IPv4 and IPv6 backend pool names.
 func getBackendPoolNames(clusterName string) map[bool]string {
 	return map[bool]string{
-		false: getBackendPoolName(clusterName, false),
-		true:  getBackendPoolName(clusterName, true),
+		IsIPv4: getBackendPoolName(clusterName, IsIPv4),
```

Comment:
can we define a enum type for IsIPv4 and IsIPv6? I'm not sure these two variables are mutually exclusive.

Reply:
IsIPv4 is actually NotIPv6, so they are mutually exclusive.